fwdpy11_arg_example/README.rst

```diff
@@ -0,0 +1,121 @@
+
+License
+----------------------------------
+
+GPLv3 or later (see COPYING)
+
+Overview
+----------------------------------
+
+We define a C++ class called "ancestry_tracker", which stores nodes and edges as they appear forwards in time.  These data structures, and their updating, are non-intrusive, meaning that they don't care about any of the fwdpp_ internals.  Rather, we simply have to define a new "iterate a generation" function that uses both fwdpp_ machinery and updates an ancestry_tracker as appropriate.
+
+Using pybind11_, we make ancestry_trackers visible to Python as an AncestryTracker class.  The Python class has access to the nodes and edges as NumPy structured arrays, which can be viewed "for free", meaning that no copy from C++ to Python is required to look at them.
+
+We define an ArgSimplifier class to bridge the AncestryTracker and msprime.  The __call__ operator of ArgSimplifier will accept an AncestryTracker as an argument and use the msprime API to simplify the input data into a set of trees.
+
+Caution
+----------------------------------
+
+This is a proof of principle implementation only.  It has been tested that it gives the correct sample properties in distribution and that internal data structures are sane during simulation.  The API is not guaranteed to be neither ideal nor idiomatic.  Further, various safety checks are missing on the C++ side.  We simply do not check for integer overflow or other possible things that may happen if garbage collection occurs too infrequently.  These limitations, and others, will be dealt with (and tested for) when we port these ideas into fwdpy11_.
+
+Usage instructions
+----------------------------------
+
+This section will get you to a point where you can run the C++ simulations described in the paper.  These instructions have been confirmed to work in a clean conda environment using Python3.  **We strongly recommend that this package is compiled within a clean conda env.**
+
+First, install a Python3 environment for your user.  It doesn't matter if you use a "full" installer or a "miniconda"
+installer.  At this point, you may need to follow the conda installation instructions_.
+
+.. note::
+
+    Linux is the preferred environment here.  If you insist on using Apple's OS X, then you
+    must add a --gcc flag to the last command in the following code block.  I (KRT) make 
+    no guarantees about success on OS X.
+
+Instructions for conda on Linux:
+
+.. code-block:: bash
+
+    conda create --name ftprime_ms python
+    source activate ftprime_ms
+    conda config --add channels defaults
+    conda config --add channels bioconda
+    conda config --add channels conda-forge
+    conda install gcc fwdpy11==0.1.4 msprime==0.5.0 pybind11==2.2.1 pandas
+    git clone https://github.com/molpopgen/fwdpy11_arg_example
```

Comment:
Delete next two lines, right?

Reply:
Or at least, the clone and 'cd' are now redundant. Could say at the start to make sure CWD is where this readme is.